<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stucell_course</class>
 <widget class="QDialog" name="stucell_course">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="findCourse"/>
     </item>
     <item>
      <widget class="QPushButton" name="showCourse">
       <property name="text">
        <string>查询课程</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectCourse">
       <property name="text">
        <string>选课</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitCourse">
       <property name="text">
        <string>退课</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="myCourse">
       <property name="text">
        <string>查看已选课程</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
